gix-url/tests/fixtures/make_baseline.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# list of urls that should be tested for all platforms
+tests=()
+# urls only intended for testing on Unix platforms
+tests_unix=()
+# urls only intended for testing on Windows
+tests_windows=()
+
+# The contents and structure of this loop are a direct copy
+# from git's own test suite (t/t5500-fetch-pack.sh).
+# Please do not change this loop and instead add additional
+# test cases at the bottom of this file.
+for path in "repo" "re:po" "re/po"
```

Comment:
Would you use `; do` instead for consistency? That's the style used so far in all existing scripts.